web/client/components/data/featuregrid/enhancers/editor.js

```diff
@@ -69,12 +72,27 @@ const featuresToGrid = compose(
                     editable: props.mode === "EDIT",
                     sortable: !props.isFocused
                 }, {
-                    getEditor: ({localType=""} = {}) => props.editors(localType, {
-                        onTemporaryChanges: props.gridEvents && props.gridEvents.onTemporaryChanges,
-                        autocompleteEnabled: props.autocompleteEnabled,
-                        url: props.url,
-                        typeName: props.typeName
-                    }),
+                    getEditor: (desc) => {
+                        // if configured it return the custom editor depending on the localtype of the present column
+                        const editorProps = {
```

Comment:
simplify. If the getCustomEditor function (with same signature of the other editor provider) returns an enditor, use it. Otherwise use old logic